src/AdminModule/Components/Sidebar/Sidebar.tsx

```diff
@@ -1,7 +1,312 @@
+import ChangePassword from '../../../AuthModule/Components/ChangePassword/ChangePassword';
+import { Link } from 'react-router-dom';
+
+import { InsertLink } from '@mui/icons-material';
+const drawerWidth = 210;
+
+const openedMixin = (theme: Theme): CSSObject => ({
+  width: drawerWidth,
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  }),
+  overflowX: 'hidden',
+});
+
+const closedMixin = (theme: Theme): CSSObject => ({
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  overflowX: 'hidden',
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up('sm')]: {
+    width: `calc(${theme.spacing(8)} + 1px)`,
+  },
+});
+
+const DrawerHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  backgroundColor:"#203FC7",
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+}));
+
+
+
+
+const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop) => prop !== 'open' })(
+  ({ theme, open }) => ({
+    width: drawerWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+    boxSizing: 'border-box',
+    ...(open && {
+      ...openedMixin(theme),
+      '& .MuiDrawer-paper': openedMixin(theme),
+    }),
+    ...(!open && {
+      ...closedMixin(theme),
+      '& .MuiDrawer-paper': closedMixin(theme),
+    }),
+  }),
+);
+
 
 export default function Sidebar() {
+
+  const navigateTo=useNavigate();
+  const theme = useTheme();
+  const [open, setOpen] = React.useState(false);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+ 
+  const [show, setShow] = React.useState(false);
+  const handleShow = () => setShow(true);
+  const handleClose = () => setShow(false);
+ 
   return (
-    <div>Sidebar</div>
-  )
+    <>
+   < Modal show={show} onHide={handleClose}>
+        <Modal.Body>
+        <ChangePassword handleClose={handleClose}/>
+        </Modal.Body>
+      </Modal>
+
+  
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
+      <Drawer variant="permanent" open={open} >
+        <DrawerHeader>
+      
+          <IconButton onClick={handleToggle}>
+            {theme.direction === 'rtl' ? <ChevronRightIcon /> : <ChevronLeftIcon />}
+          </IconButton>
+        </DrawerHeader>
+        <Divider />
+        <List sx={{backgroundColor:"#203FC7",color:"white"}}>
+
+            <ListItem  disablePadding sx={{ display: 'block' }}>
+              <ListItemButton
+              onClick={()=>{navigateTo("/Admin/home")}}
+                sx={{
+                  minHeight: 48,
+                  justifyContent: open ? 'initial' : 'center',
+                  px: 2.5,
+                }}
+              >
+                <ListItemIcon
+                  sx={{
+                    color:"white",
```

Comment:
We should be using the colors in theme, this will help us later if we want to update theme or add dark/light mode functionality.